public/js/graph/react_graph.js

```diff
@@ -895,19 +921,55 @@ var EdgeGroup = React.createClass({
                      source={edgeJSON.source}
                      target={edgeJSON.target}
                      points={edgeJSON.points}
-                     svg={this.props.svg} />;
+                     svg={this.props.svg}
+                     edgeID={edgeJSON.id_}
+                     updateEdgeStatus={this.updateEdgeStatus} />;
     },
 
     render: function () {
+        // Missing edges must be rendered last. The sort
+        // method custom sorts a copy of edgesJSON so that all missing edges 
+        // are last in the list. Then render based on that list.
+        var edges = this.props.edgesJSON;
+        var edgesCopy = $.extend([], edges);
+        var state = this.state;
+        edgesCopy.sort(function(a, b) {
+            // If an edge is missing, its edgeID should be in EdgeGroup's
+            // state and its value should be true.
+            var aID = a.id_;
+            var bID = b.id_;
+            var aMiss = false;
+            var bMiss = false;
+            if (aID in state) {
+                if (state[aID]) {
```

Comment:
I understand now that this is a boolean. You should combine this check with the previous one (`aID in state && state[aID]`), and also use it to set `aMiss` directly, rather than using an `if` (similar to above comment). Same with `bMiss`.